service/src/main/kotlin/no/nav/su/se/bakover/service/kontrollsamtale/KontrollsamtaleService.kt

```diff
@@ -55,7 +67,15 @@ class KontrollsamtaleServiceImpl(
 
         return Either.catch {
             sessionFactory.withTransactionContext {
+                val kontrollsamtale = Kontrollsamtale.om4Måneder(
+                    sakId = sakId,
+                    fraDato = LocalDate.now(clock),
+                    stønadsperiodeslutt = gjeldendeStønadsperiode.tilOgMed,
+                )
                 brevService.lagreDokument(dokument, it)
```

Comment:
Dette vil vel føre til at brevet sendes ut umiddelbart?

Reply:
Her har vi vel muligens et behov for å utvide eksisterende funksjonalitet for journalføring av brev/utsending av brev med "tidskontrollert utsending" - f.eks et felt som forteller jobben for distribusjon at dette brevet "tidligst kan sendes ut fra og med dato x" eller lignende.

Reply:
Er det hensiktsmessig at brevet som skal sendes ut om 4 måneder produseres nå, eller bør dette utsettes til om 4 måneder? Mest i forhold til å få med oppdaterte endringer i brevene f.eks.

Reply:
Denne var litt forvirrende, så vi trekker ut planleggingen av "neste kontrollsamtale" til en funksjon slik at det leser litt bedre. Det som skjer her er jo egentlig 2 ting:
- Umiddelbar innkalling til kontrollsamtale (som lager og sender brev)
- Planlegging av neste kontrollsamtale (som ikke lager brev)